From 37a8c69b729be0d78e65a79a5f70b690b8443f24 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:45 -0400
Subject: [redhat] kabi: Adding symbol device_register (drivers/base/core.c)

Message-id: <931df796a536656c380cc9cdb462696c86eb5672.1412771859.git.poros@redhat.com>
Patchwork-id: 97178
O-Subject: [RHEL7.1 PATCH 25/67] kabi: Adding symbol device_register (drivers/base/core.c)
Bugzilla: 1148046
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148046
    https://bugzilla.redhat.com/show_bug.cgi?id=1148046

    Description:
    ============
    This is used with low level character devices created
    by the extint.k, ioc4_extint.ko, and pcie_rt.ko modules. device_unregister is
    already whitelisted.
    Symbol used in extint.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/device_register
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/device_register
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/device_register

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/device_register b/redhat/kabi/kabi-module/kabi_ppc64/device_register
new file mode 100644
index 0000000..8339ee9
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/device_register
@@ -0,0 +1,2 @@
+#1-
+0xd086f483 device_register vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/device_register b/redhat/kabi/kabi-module/kabi_s390x/device_register
new file mode 100644
index 0000000..17df60b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/device_register
@@ -0,0 +1,2 @@
+#1-
+0xc1029a77 device_register vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/device_register b/redhat/kabi/kabi-module/kabi_x86_64/device_register
new file mode 100644
index 0000000..f385539
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/device_register
@@ -0,0 +1,2 @@
+#1-
+0xf283da2a device_register vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1